### PR TITLE
[ELY-1479] Coverity, Missing call to superclass in AbstractGssapiMechanism

### PR DESCRIPTION
**Pull request by ivassile**
wildfly-elytron: https://issues.jboss.org/browse/ELY-1479
JBEAP: https://issues.jboss.org/browse/JBEAP-14149

### CODE REVIEW

File: src/main/java/org/wildfly/security/sasl/gssapi/AbstractGssapiMechanism.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ public Object getNegotiatedProperty(String propName) {
                 return Integer.toString(maxBuffer);
         }
 
-        return null;
+        return super.getNegotiatedProperty(propName);
```
Comment (dmlloyd) [inline src/main/java/org/wildfly/security/sasl/gssapi/AbstractGssapiMechanism.java]: The superclass only repeats the assertComplete() call.  I don't think this PR adds any value.

Comment (ivassile) [reply src/main/java/org/wildfly/security/sasl/gssapi/AbstractGssapiMechanism.java]: I agree that the current implementation doesn’t change the behaviour but from a design point of view it unifies with the rest of the subclasses. Theoretically the behavior will change with future superclass changes.

**Comment by darranl** [comment]
Maybe the other two mechanisms that triggered this to be flagged up should just be updated to return null instead so we skip the double call to assertComplete.

In this case the parent implementation is just to remove the need for the child to implement the method if not really required, a call to the parent is not really needed as the child is taking full control.

**Comment by ivassile** [comment]
@darranl Implemented requested changes.

**Pull request merged**